metrics/integration_test.go

```diff
@@ -33,8 +33,6 @@ import (
 var _ = Describe("Metrics Integration", Ordered, func() {
 	BeforeAll(func() {
 		metrics.Registry.Unregister(SnapshotCreatedToPipelineRunStartedSeconds)
-		metrics.Registry.Unregister(SnapshotConcurrentTotal)
-		metrics.Registry.Unregister(SnapshotDurationSeconds)
```

Comment:
Is this removal right? Shouldn't we keep SnapshotDurationSeconds here?

Reply:
It's right from the test.

Reply:
What does it mean "right from the test"?

I can write passing unittest for any broken code.

How `SnapshotDurationSeconds` is related to Concurent Snapshot change?

Reply:
It can be restored.

Reply:
I'd like to have it restore then